OSDOCS-12080: Documented the 4.17 RN for the nmstate Operator metrics (#82212)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator_k8s-nmstate-operator[About the Kubernetes NMState Operator].

[id="ocp-4-17-networking-nmstate-operator-metrics_{context}"]
==== View metrics collected by the Kubernetes NMState Operator

> The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. Both *Administrator* and *Developer* perspectives support viewing these metrics.

Reviewer: ```suggestion
The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using  the *Administrator* and *Developer* perspectives.
```

Author: :heavy_check_mark:

> The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using  the *Administrator* and *Developer* perspectives.

Reviewer: ```suggestion
The Kubernetes NMState Operator, `kubernetes-nmstate-operator`, can collect metrics from the `kubernetes_nmstate_features_applied` component and expose them as ready-to-use metrics. You can view these metrics by using the *Administrator* and *Developer* perspectives.
```
Sorry, my suggested change too had an extra space, which needs to be removed.

Author: Thanks. Updated!
